src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java

```diff
@@ -35,23 +39,26 @@ public void setName(final String newName) {
 
     /**
      * Gets frame select of video.
-     * @return frame is value select options.
+     *
+     * @return a String with frame is value select options.
      */
     public String getFrame() {
         return frame;
     }
 
     /**
      * Sets value frame select of video.
+     *
      * @param newFrame is value select options.
      */
     public void setFrame(final String newFrame) {
         this.frame = newFrame;
     }
 
     /**
-     * Gets string select of combBox.
-     * @return string is value displayed in combBox.
+     * Gets string select of comboBox.
+     *
+     * @return a String is value displayed in comboBox.
```

Comment:
It would be better to put: "a String with the displayed value on the comboBox"

Reply:
ok ray, i correct it